assets/chezmoi.io/docs/reference/templates/directives.md

```diff
@@ -0,0 +1,46 @@
+# Directives
+
+## Delimiters
+
+By default, chezmoi uses the standard `text/template` delimiters `{{` and `}}`.
+If a template contains the string
+
+    chezmoi:template:left-delimiter=$LEFT right-delimiter=$RIGHT
+
+Then the delimiters `$LEFT` and `$RIGHT` are used instead. Either or both of
+`left-delimiter=$LEFT` and `right-delimiter=$RIGHT` may be omitted. `$LEFT` and
+`$RIGHT` must be quoted if they contain spaces. If either `$LEFT` or `$RIGHT`
+is empty then the default delimiter (`{{` and `}}` respectively) is set
+instead.
+
+chezmoi will remove the line containing `chezmoi:template:` directives to
```

Comment:
This paragraph mostly belongs just below `# Directives`, removing the `to avoid parse errors from the delimiters`. That’s one reason, but the other would be `to avoid affecting the syntax of the resulting template`.

Reply:
Thanks, done.